modules/apps/callbacks/testing/simapp/app.go

```diff
@@ -511,7 +511,11 @@ func NewSimApp(
 	var transferStack porttypes.IBCModule
 	transferStack = transfer.NewIBCModule(app.TransferKeeper)
 	transferStack = ibccallbacks.NewIBCMiddleware(transferStack, app.IBCFeeKeeper, app.MockContractKeeper, maxCallbackGas)
-	transferICS4Wrapper := transferStack.(porttypes.ICS4Wrapper)
+	transferICS4Wrapper, ok := transferStack.(porttypes.ICS4Wrapper)
+	if !ok {
+		panic("Can't convert transferStack to ICS4Wrapper")
```

Comment:
I think you could do here also something like...

```suggestion
		panic(fmt.Errorf("cannot convert %T to %T", transferStack, porttypes.ICS4Wrapper))
```

Reply:
Done, but as in all similar occurrences, I had to use an extra variable as passing the type directly (or `&porttypes.ICS4Wrapper{}` won't work. If there's a way to achieve this that I missed, please let me know! :)